app.js

```diff
@@ -61,6 +63,15 @@ process.chdir(__dirname);
     }
   }
 
+  // In dev mode, ensure unhandles promises log to the console with the full stack
+  // See https://stackoverflow.com/a/46661020
+  if (!isProd) {
+    process.on('unhandledRejection', (err) => {
+      console.error(err);
+      throw err;
+    });
+  }
```

Comment:
This was the key to debugging that unhandled promise. This will force it to console log as the full stack trace. 😎